stream-chat-android-client/src/test/java/io/getstream/chat/android/client/chatclient/WhenReconnectSocket.kt

```diff
@@ -79,13 +79,6 @@ internal class WhenReconnectSocket : BaseChatClientTest() {
         verify(socket).reconnectUser(user, isAnonymous = true)
     }
 
-    @Test
-    fun `Given disconnected connection state And anonymous pending state Should throw exception`() {
-        val sut = Fixture().givenDisconnectedConnectionState().givenAnonymousPendingState().get()
-
-        invoking { sut.reconnectSocket() }.shouldThrow(IllegalStateException::class)
-    }
-
```

Comment:
Hey @amitkma,

I wonder why this test case got deleted

Reply:
Cause we don't have anonymous pending state anymore.